components/gitpod-protocol/src/util/gitpod-host-url.ts

```diff
@@ -138,6 +138,10 @@ export class GitpodHostUrl {
         return this.url.toString();
     }
 
+    toRelative(): string {
```

Comment:
This is needed because the `<Link>`s do not support full-blown URLs, just relative ones.